tests/test_deploy.py

```diff
@@ -24,11 +24,11 @@ def wellcome_project_file(tmpdir, project_id, role_arn):
               image_repositories:
                 - id: repo1
                   services:
-                  - service1a
-                  - service1b
+                  - id: service1a
```

Comment:
Will this mean that consumers need to update their `.wellcome_project` files?

Reply:
Good question, but no. Existing .wellcome_project files already use the correct format here; this is wrong but was never noticed previously.